tests/packages/link-library-in-subproject/pyproject.toml

```diff
@@ -0,0 +1,17 @@
+# SPDX-FileCopyrightText: 2022 The meson-python developers
+#
+# SPDX-License-Identifier: MIT
+
+[build-system]
+build-backend = 'mesonpy'
+requires = ['meson-python']
+
+[project]
+name = 'link-library-in-subproject'
+version = '1.2.3'
+license = 'MIT'
```

Comment:
This makes the test dependent on pyproject-metadata 0.9.0 (the test fails on the CI job that tests with pyproject-metadata 0.8.0, which is currently the minimum required version)

Reply:
Yes, seeing the failure - I'll drop this line then.

Reply:
In #681 we are updating the minimum required version to 0.9.0. I would kip this.

Reply:
Removed this in the end; it served no real purpose and almost all other test packages don't have a `license =` entry.